app/backend/src/couchers/notifications/settings.py

```diff
@@ -129,6 +129,8 @@ def set_preference(session, user_id, topic_action: NotificationTopicAction, deli
                     ("create_approved", "An event that is approved by the moderators is created in your community"),
                     ("create_any", "A user creates any event in your community (not checked by an admin)"),
                     ("update", "An event you are attending is updated"),
+                    ("cancel", "An event you were attending has been cancelled"),
+                    ("delete", "An event you were attending has been deleted"),
```

Comment:
"An event you are attending is cancelled/deleted"

Reply:
Ah, I misunderstood these settings - changed it 👍